### PR TITLE
Readiness and Liveness reports

### PR DESCRIPTION
**Pull request by jsenko**
Run the app and then `curl -v http://localhost:8080/health`

The commit also contains some minor automatic formatting changes, I can remove them if needed. Maybe we could decide on a common code style.

### CODE REVIEW

File: app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        IGNORED.add(RuleNotFoundException.class);
+        IGNORED.add(VersionNotFoundException.class);
+    }
+
+    @Inject
+    PersistenceExceptionLivenessCheck check;
+
+    @AroundInvoke
+    public Object intercept(InvocationContext context) throws Exception {
+        log.warn("FUUUUUUUUUUCKKCKCKCKC");
```
Comment (jsenko) [inline app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java]: Ooops

Comment (EricWittmann) [inline app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java]: Maybe remove this... :)

File: app/src/main/java/io/apicurio/registry/metrics/PersistenceSimpleReadinessCheck.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+     */
+    private boolean test() {
+        try {
+            List<RuleType> res = storage.getGlobalRules();
+            if (res == null) {
+                return false;
+            }
+            return true;
+        } catch (Exception ex) {
+            ex.printStackTrace();
```
Comment (EricWittmann) [inline app/src/main/java/io/apicurio/registry/metrics/PersistenceSimpleReadinessCheck.java]: Use the logger here.

File: app/src/main/java/io/apicurio/registry/metrics/ResponseTimeoutReadinessCheck.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import javax.ws.rs.ext.Provider;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+/**
+ * @author Jakub Senko <jsenko@redhat.com>
+ */
+@ApplicationScoped
+@Readiness
```
Comment (EricWittmann) [inline app/src/main/java/io/apicurio/registry/metrics/ResponseTimeoutReadinessCheck.java]: Should this be liveness or readiness?

File: app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java (hunk 1 above)
Comment (jsenko) [reply app/src/main/java/io/apicurio/registry/metrics/PersistenceExceptionLivenessInterceptor.java]: Yeah, I could not get the interceptors to work and got carried away... oops

**Comment by EricWittmann** [comment]
I tried building this locally and got the same error as what's on CI.

**Pull request merged**